addons/escoria-core/game/core-scripts/esc/esc_event_manager.gd

```diff
@@ -278,7 +278,7 @@ func queue_background_event(channel_name: String, event: ESCEvent) -> void:
 	events_queue[channel_name].append(event)
 
 
-# Interrupt the events currently running and any that are pending.
+# interrupt() the events currently running and any that are pending.
```

Comment:
Uh oh...I think your use of sed was a little overzealous. ;) Comments and error messages that now read "interrupt()" instead of "Interrupt" or "Interrupting", etc., should be put back. Sorry!

Reply:
Fixed! Sorry about that.